### ISSUE TITLE
Object is not abstract and does not implement abstract member public abstract fun error

### ISSUE THREAD

**Issue by saileshbro** [infinum/flutter-prince-of-versions#14]
Unable to use the plugin with the latest Flutter 3.
```
e: /Users/saileshbro/.pub-cache/hosted/pub.dartlang.org/flutter_prince_of_versions-2.0.1/android/src/main/kotlin/com/example/flutter_prince_of_versions/FlutterPrinceOfVersionsPlugin.kt: (117, 79): Object is not abstract and does not implement abstract member public abstract fun error(p0: String, p1: String?, p2: Any?): Unit defined in io.flutter.plugin.common.MethodChannel.Result
e: /Users/saileshbro/.pub-cache/hosted/pub.dartlang.org/flutter_prince_of_versions-2.0.1/android/src/main/kotlin/com/example/flutter_prince_of_versions/FlutterPrinceOfVersionsPlugin.kt: (121, 33): 'error' overrides nothing
2

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_prince_of_versions:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
Exception: Gradle task assembleDevDebug failed with exit code 1
```

**Comment by saileshbro** [comment]
#15

**Comment by KhaleelSH** [comment]
Closing the issue as it's fixed in #15 .
